httplogging/amqp.go

```diff
@@ -62,23 +63,26 @@ func (c *AmqpLoggingBehaviour) connect() error {
 			<-timer.C
 
 			c.logger.Errorf("reconnecting after connection error")
-			c.connect()
+			err := c.connect()
+			if err != nil {
+				c.logger.Errorf("reconnecting after connection error failed")
```

Comment:
Also log the actual error:

```suggestion
				c.logger.Errorf("reconnecting after connection error failed: %s", err.Error())
```

Reply:
Done